app/Services/PolygonService.php

```diff
@@ -45,6 +49,58 @@ class PolygonService
         'num_trees',
     ];
 
+    public function createProjectPolygon($entity, $currentGeojson)
+    {
+        if (GeometryHelper::isFeatureCollectionEmpty($currentGeojson)) {
+            return;
+        }
+
+        $needsVoronoi = GeometryHelper::isOneOrTwoPointFeatures($currentGeojson);
+        if ($needsVoronoi) {
+            $pointWithEstArea = GeometryHelper::addEstAreaToPointFeatures($currentGeojson);
+            $currentGeojson = App::make(PythonService::class)->voronoiTransformation(json_decode($pointWithEstArea));
+        }
+
+        $convexHullWkt = GeometryHelper::getConvexHull($currentGeojson);
+        if ($convexHullWkt) {
```

Comment:
I don't think that the `Wkt` suffix is anything useful on the variable name.

Reply:
Done.